### PR TITLE
Ref/elastica migration part3

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |Y|
|New feature?   |Y|	
|BC breaks?     |N|	
|Deprecations?  |Y|	
|Fixed tickets  |N|

### CODE REVIEW

File: src/Elasticsearch/Bulker.php [hunk 1]
```diff
@@ -96,53 +71,47 @@ public function setSize(int $size): Bulker
...
+            $action->setSource($body);
         }
-
+        $this->bulk->addAction($action);
         ++$this->counter;
 
         return $this->send();
     }
 
-    public function indexDocument(DocumentInterface $document, string $index, bool $upsert = false): bool
```
Comment (Davidmattei) [inline src/Elasticsearch/Bulker.php:132]: upsert removed, please put it back

File: src/Twig/AppExtension.php [hunk 2]
```diff
@@ -5,10 +5,12 @@
 use Caxy\HtmlDiff\HtmlDiff;
 use DateTime;
 use Doctrine\Bundle\DoctrineBundle\Registry;
-use Elasticsearch\Client;
+use Elastica\ResultSet;
```
Comment (Davidmattei) [inline src/Twig/AppExtension.php:8]: Don't we need to use the common result set? 

https://github.com/ems-project/EMSCommonBundle/blob/develop/src/Elasticsearch/Elastica/ResultSet.php

**Review by Davidmattei — DISMISSED** [review]

File: src/Twig/AppExtension.php (hunk 2 above)
Comment (theus77) [reply src/Twig/AppExtension.php:8]: It's an hook to make the client 5 working with es7

File: src/Elasticsearch/Bulker.php (hunk 1 above)
Comment (theus77) [reply src/Elasticsearch/Bulker.php:132]: Done

File: src/Twig/AppExtension.php (hunk 2 above)
Comment (Davidmattei) [reply src/Twig/AppExtension.php:8]: okay aggree

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**